sharkfuser/benchmarks/run_benchmark.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import csv
+import glob
+import os
+import shlex
+import statistics
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+from typing import NamedTuple
+
+
+class TimingStats(NamedTuple):
+    min: float | str = "N.A."
+    max: float | str = "N.A."
+    mean: float | str = "N.A."
+    stddev: float | str = "N.A."
+    count: int | str = "N.A."
+
+
+ALL_STATS = ["min", "max", "mean", "stddev", "count"]
```

Comment:
Please add another column for `dispatch_count` (this is useful info to look at a glance), and consider renaming `count` --> `iter` to match with the CLI flag. 
By `dispatch_count`, I mean just 1 for single dispatch, 2 for two dispatches, and not 2 * iter.